### DISCUSSION TITLE
Fake input field

### DISCUSSION THREAD

**Discussion by JoseGonzalz** [Telefonica/mistica-design#1757] [Proposals]
### Goal

We need to create an entry point with a fake input field for the search engine project.

### Context

Figma: https://www.figma.com/file/q1xa2UZwbYttRO6NNZqx9L/Search-Engine_VIVOBR?type=design&node-id=838-60851&mode=design&t=2f0uoMNPlNb5jKTp-4

### Impact

Creating a better user experience for this project.

### Discussion Questions

CMS team need to coordinate with Mística on how to do this component and its behaviour.

**Comment by yceballost** [comment]
We can see some ways to go:

## Creating an extended component for this
We think that this is not the best approach because you have to maintain the UI equivalence with the original inputs along the time

## Making a fake input field in Mística
We will absolutely dismiss this way if it's possible.

## Adding a `fake` prop to inputs fields
We don't really like this approach because we can get to fill all the components with this prop. IMPO, this is not a good practice at all.

## Forcing no-interaction in input
Would be possible to wrap the component in a div with certain properties that remove interactions and accessibility properties to make the element invisible in terms of interactions and accessibility and then wrap this div and input in a `<Touchable>` component that provide interaction and accessibility label (this accessibility label must be defined by product design). That's a quick approach from @pladaria 

[Here the example](https://mistica-web.vercel.app/playroom#?code=N4Igxg9gJgpiBcIA8AVCBXMALAhgIwBsYAdAOwAJyBnAFwE8iBeYYcvCAJ1g-nOJACMABwAe5KDipYYUckICWpANb8ANOQDu8qDSy9%2BAM3k0AtJFI0YF-uQC%2BtspQikAChxhUqzABQBKcowAfOTAjpTUMDQAyjQ4lt78YGrkAOaRMXEwCeDJAAz%2BANTkAr4A3GH2ZIFhSFDyAG7kOBzyOCZY2rCkjPw0HOgkINT0TCxyEIqWHACi9VY0VPogpM6DdrbVFOFIUTDN2ABi8jAEsqQ4ALYwPSC7%2B1g2BPgnN1R7HNj8gUgA9HcfWCOJygm0ovzq9U2vzQmFwhBgmzItQagTcHjesmAaWisXiiTyvls5Bo8iuVHBKLIIFUIF0MDJCAA2iAALIQeryWjNEAAXRpWh0WCoTIAzAAmXKqAS5MUAFh5tiAA)

In any case, the web team will think of an alternative to this. I created a ticket to keep it mind → WEB-1858

**Comment by yceballost** [comment]
@JoseTelefonica can we close this discussion?

**Reply by JoseGonzalz** [reply]
Yes, no problem. Tnks!
